projects/commudle-admin/src/app/feature-modules/public-community/components/speakers/speakers.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="speakers-container d-flex flex-wrap justify-content-center ">
+  <nb-card *ngFor="let speaker of user">
+    <a class="clickable" [routerLink]="['/users', speaker.username]">
```

Comment:
There should be no tag between nb-card and nb-card-body. Please look at the documentation in nebular components